xbmc/cores/VideoPlayer/DVDCodecs/Video/DXVA.cpp

```diff
@@ -71,6 +83,9 @@ static const int PROFILES_VP9_0[]        = { FF_PROFILE_VP9_0,
                                              FF_PROFILE_UNKNOWN };
 static const int PROFILES_VP9_10_2[]     = { FF_PROFILE_VP9_2,
                                              FF_PROFILE_UNKNOWN };
+static const int PROFILES_AV1_MAIN[] = {FF_PROFILE_AV1_MAIN, FF_PROFILE_UNKNOWN};
```

Comment:
This seems to be just "following the pattern" but what about using constexpr in an anonymous namespace to keep up with the code guidelines and to remove the static usage?